cmd/ak/cmd/sessions/delete.go

```diff
@@ -0,0 +1,58 @@
+package sessions
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+)
+
+var deleteCmd = common.StandardCommand(&cobra.Command{
+	Use:     "delete [session ID] [--fail]",
+	Short:   "Delete non-running session",
+	Aliases: []string{"d"},
+	Args:    cobra.MaximumNArgs(1),
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
```

Comment:
A point for discussion: with other session commands, considering the latest session as a reasonable and desirable default makes sense. With `delete`, given that it's destructive, should we allow an implicit default at all, or should we force the user to specify the ID for the sake of safety? (i.e. `Args: cobra.ExactArgs(1)` above)

Reply:
inviting @haimzlato to the discussion as well.

1. I think the goal of having good CLI is to facilitate user interaction with the system. Specifying sessionID (which requires copy/paste) is a bit annoying IMO. So, I suggest to allow user to delete last session without specifying explicitly the sessionID (similar to how stop CMD is implemented)
2. actually, delete command isn't that destructive. We are marking sessions as deleted in DB and allowing to do so only for non-running sessions. Moreover, session being deleted is printed to stdout, so user have a copy in case he deleted not what he  wanted
3. Although we may consider to allow user to delete implicitly only sessions resulting in ERROR and require explicit sessionID for those ended with SUCCESS

Reply:
I think allowing any action without a specific id is problematic

in a real production system, session could start at any time and on many projects
delete \ get logs of a non specific value would in high probability result unintended stuff

it's not so hard to copy paste an id once

Reply:
agree with @efiShtain 
sessionID is now required